script.js

```diff
@@ -0,0 +1,65 @@
+let Books = [];
+
+// Book Div and Template const
+const temp = document.querySelector('.book');
+const bookshelf = document.querySelector('#bookshelf');
+
+const submit = document.querySelector('#submit');
+let idBook = Books.length;
+
+function Book(title, author) {
+  this.id = idBook;
+  this.title = title;
+  this.author = author;
+  idBook += 1;
+}
+
+function DeleteBook(id) {
+  Books = Books.filter((book) => book.id !== id);
+  localStorage.Books = JSON.stringify(Books);
+  Books = JSON.parse(localStorage.Books);
+  bookshelf.innerHTML = '';
+  bookshelf.appendChild(temp);
+}
```

Comment:
- [ ] When I try to delete a book, all of them are removed from the page, and I have to reload the page in order for them to update correctly.  Please, correct this behavior so that when a book is deleted that one is only deleted from the page, without having to reload it.

## Explanation

When I remove a book, only that book should be removed from the DOM, not all of them.  There should be no need to reload the page in order for the books to display again.

## Hints

- You could generate an HTML ID for each book, instead of having just one ID for all the books.
- You could use the book author and/or the title as the id to remove them.

- [ ] After playing with the app, this happens:

![image](https://user-images.githubusercontent.com/57421823/163958123-ec232eea-a0d1-4cd0-8d50-6d346cc468d4.png)

As you can see, the IDs on localStorage sometimes get repeated.  When this happens, it is impossible to delete just a single book.

Please, kindly correct this behavior.

That way, your app will work as intended! 💪